### PR TITLE
Add tests for Microsoft.Data.SqlClient ver 1.0.19239.1

### PR DESCRIPTION
**Pull request by vuqtran88**
## Description
Add tests for Microsoft.Data.SqlClient ver 1.0.19239.1

### CODE REVIEW

File: tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj [hunk 1]
```diff
@@ -32,10 +32,10 @@
 
 
     <!--Microsoft.Data.SqlClient-->
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.0.1" Condition="'$(TargetFramework)' == 'net462'" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="1.0.19239.1" Condition="'$(TargetFramework)' == 'net462'" />
     <PackageReference Include="Microsoft.Data.SqlClient" Version="5.0.1" Condition="'$(TargetFramework)' == 'net471'" />
```
Comment (chynesNR) [inline tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj:36]: Should we try a couple other versions in here as well?

File: tests/Agent/IntegrationTests/UnboundedIntegrationTests/MsSql/MsSqlAsyncParameterizedQueryTests.cs [hunk 2]
```diff
@@ -234,6 +234,33 @@ public MicrosoftDataSqlClientAsyncTestsParameterizedQuery_ParamsWithoutAtSignFra
         }
     }
 
+    [NetFrameworkTest]
```
Comment (jaffinito) [inline tests/Agent/IntegrationTests/UnboundedIntegrationTests/MsSql/MsSqlAsyncParameterizedQueryTests.cs:237]: The naming of this test class is inconsistent with other similar classes.  "Framework" should be abbreviated as "FW" in the class similar to how it is in the fixture or ConsoleMF class.

File: tests/Agent/IntegrationTests/UnboundedIntegrationTests/MsSql/MsSqlStoredProcedureTests.cs [hunk 3]
```diff
@@ -162,6 +162,32 @@ public MicrosoftDataSqlClientStoredProcedureTests_ParamsWithoutAtSignsFramework(
         }
     }
 
+    [NetFrameworkTest]
```
Comment (jaffinito) [inline tests/Agent/IntegrationTests/UnboundedIntegrationTests/MsSql/MsSqlStoredProcedureTests.cs:165]: Same naming issue.

File: tests/Agent/IntegrationTests/UnboundedIntegrationTests/MsSql/MsSqlWithParameterizedQueryTests.cs [hunk 4]
```diff
@@ -198,6 +198,34 @@ public MicrosoftDataSqlClientQueryParameterCaptureTests_ParamsWithoutAtSignsFram
         }
     }
 
+    [NetFrameworkTest]
+    public class MicrosoftDataSqlClientQueryParameterCaptureTestsFramework462 : MsSqlQueryParameterCaptureTestsBase<ConsoleDynamicMethodFixtureFW462>
```
Comment (jaffinito) [inline tests/Agent/IntegrationTests/UnboundedIntegrationTests/MsSql/MsSqlWithParameterizedQueryTests.cs:202]: Same naming issue.

File: tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj (hunk 1 above)
Comment (vuqtran88) [reply tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj:36]: We could but I don't have a strong reference on what other versions we should test. Could you give a list ?

Comment (chynesNR) [reply tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj:36]: In general I think covering as many major versions as possible is a good way to do it. It looks like there's a 2.1.5, a 3.1.1, and a 4.1.1.

Comment (vuqtran88) [reply tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj:36]: ok. I will update with these versions.

**Review by chynesNR — APPROVED** [review]

**Pull request merged**